src/modules/settings/Options.js

```diff
@@ -12,7 +12,10 @@ export default class Options {
       seriesName: undefined,
       opposite: false,
       reversed: false,
-      logarithmic: false,
+      logarithmic: {
```

Comment:
The modified property is a significant change in the API and will break the log scale for all the people who use the old versions.
I propose another way to introduce this new feature. Instead of converting the logarithmic property from `boolean` to `object`, let's keep it is as, and add a new property logBase.

So, it becomes like this

```js
logarithmic: false,
logBase: 10
```

This way, we don't introduce a breaking change.